### ISSUE TITLE
Pretrained backbone

### ISSUE THREAD

**Issue by rvandeghen** [LiheYoung/UniMatch#32]
Hi,

Great work ! I was wondering where your pretrained backbone come from ?

Renaud

**Comment by LiheYoung** [comment]
I am not 100% sure, but it should come from [here](https://github.com/zhanghang1989/PyTorch-Encoding/blob/master/encoding/models/model_store.py).

**Comment by rvandeghen** [comment]
Ok thanks. In fact, I'm working on SSSS and I was simply trying to reproduce your supervised results on VOC12 based on [detectron2](https://github.com/facebookresearch/detectron2/tree/main/projects/DeepLab) and I'm far behind. I trained your supervised model with their pretrained weights and I achieved lower results, but still higher than mine. I'm still trying to understand what differs from both implementation.
